Src/IronPython/Runtime/PythonContext.cs

```diff
@@ -288,12 +288,23 @@ public PythonContext(ScriptDomainManager/*!*/ manager, IDictionary<string, objec
                     }
 #else
                     if(!IronPython.Runtime.ClrModule.IsNetCoreApp && Environment.OSVersion.Platform == PlatformID.Unix) {
-                        if(Directory.Exists("/usr/lib/ironpython2.7")) {
-                            path.append("/usr/lib/ironpython2.7");
-                        }
+                        if(IronPython.Runtime.ClrModule.IsMacOS) {
```

Comment:
What about using `if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))` and `else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)` and getting rid of `Environment.OSVersion.Platform == PlatformID.Unix`? We could probably get rid of the `IsNetCoreApp` check as well, not sure that it's all that useful.

Reply:
Yeah, that's a good idea. I keep forgetting about RuntimeInformation.